File: answer.py
def divisors(integer):
    pass 
    arr = []
    for i in range(integer):
        if i==0 or i==1 or i==integer:
            continue
        arr.append(i) if integer%i==0 else arr
    return arr if arr != [] else f'{integer} is prime'

def divisors(num):
    l = [a for a in range(2,num) if num%a == 0]
    if len(l) == 0:
        return str(num) + " is prime"
    return l

def divisors(n):
    return [i for i in xrange(2, n) if not n % i] or '%d is prime' % n

File: answer.js
function divisors(integer) {
  let newArr = []
  for(let i=0;i<=integer;i++){
    if(i!==0 && i!==1 && i!=integer ){
      integer%i==0 ? newArr.push(i) : newArr
    }
    else{
      newArr
    }
  }
  return newArr.length!==0 ?newArr: `${integer} is prime`
};

function divisors(integer) {
  var res = []
  for (var i = 2; i <= Math.floor(integer / 2); ++i) if (integer % i == 0) res.push(i);
  return res.length ? res : integer + ' is prime'
};

function divisors(integer) {
	var divs = [];
  
  for(var i = 2; i < integer; i++) {
    if(integer % i === 0) {
    	divs.push(i);
    }
  }
  
  return divs.length ? divs : integer + ' is prime';
};
